### DISCUSSION TITLE
原来Python实现HDR截图这么麻烦...

### DISCUSSION THREAD

**Discussion by 5o1** [CHNZYX/Auto_Simulated_Universe#271] [Ideas]
查了一圈, 是因为windows api不支持大于8bits颜色格式的截图, 要截HDR图片就要用OpenGL或者DirectX, 但是Python现在对DirectX的支持还是荒野, 几乎找不到现成的工具. 如果用Python写, 要么调dll, 要么用C++包装一层再调dll.
OBS里有"强制使用标准动态范围(SDR)", 查了一下源代码, 似乎是用[这个工具](https://learn.microsoft.com/en-us/uwp/api/windows.graphics.capture.direct3d11captureframepool?view=winrt-22621)实现的, 同样也是DirectX.
大家似乎或多或少都有截屏HDR的需求, 但是没人想去造轮子的样子.

**Comment by Acreg** [comment]
这也说明作者团队是真的牛X
